bindings/python/src/datetime.cpp

```diff
@@ -55,6 +55,29 @@ struct time_duration_to_python
     }
 };
 
+struct time_point_to_python
+{
+    static PyObject* convert(lt::time_point pt)
+    {
+        using boost::chrono::system_clock;
+        using boost::chrono::duration_cast;
+        time_t const tm = system_clock::to_time_t(system_clock::now()
+            + duration_cast<system_clock::duration>(pt - lt::clock_type::now()));
```

Comment:
I can't see anything wrong with this. maybe looking at ``duration_cast<lt::seconds>(pt - lt::clock_type::now()).count()`` might be useful. that should be some relatively small number of seconds (likely less then 1800 at least). If that works, maybe it would make sense to cut out ``system_clock`` and just do:

``time_t const tm = time(NULL) + duration_cast<lt::seconds>(pt - lt::clock_type::now()).count()``